### PR TITLE
vertical bar chart in V9

### PR DESCRIPTION
**Pull request by Anush2303**
## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx [hunk 1]
```diff
@@ -1,16 +1,1007 @@
...
+import { select as d3Select } from 'd3-selection';
+import {
+  scaleLinear as d3ScaleLinear,
+  ScaleLinear as D3ScaleLinear,
+  scaleBand as d3ScaleBand,
+  scaleUtc as d3ScaleUtc,
+} from 'd3-scale';
+import { useId } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import { IProcessedStyleSet, IPalette } from '@fluentui/react/lib/Styling';
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:14]: remove these imports

Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:14]: Okay but it is required in styling ?

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.types.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { IStyle } from '@fluentui/react/lib/Styling';
-import { IRenderFunction, IStyleFunctionOrObject } from '@fluentui/react/lib/Utilities';
+import { string } from '@fluentui/react/lib/Styling';
+import { IRenderFunction, stringFunctionOrObject } from '@fluentui/react/lib/Utilities';
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.types.ts:2]: remove imports

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/useVerticalBarChartStyles.styles.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  barLabel: {
+    fontSize: FontSizes.small,
+    fontWeight: FontWeights.semibold,
+    //fill: neutrals.neutralPrimary,
+  },
+});
+
+/**
+ * Apply styling to the Carousel slots based on the state
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/VerticalBarChart/useVerticalBarChartStyles.styles.ts:41]: fix typo

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/useVerticalBarChartStyles.styles.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ */
+export const useVerticalBarChartStyles_unstable = (props: IVerticalBarChartProps): IVerticalBarChartStyles => {
+  const baseStyles = useStyles();
+
+  return {
+    xAxisTicks: [],
+
+    yAxisTicks: [
+      {
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/VerticalBarChart/useVerticalBarChartStyles.styles.ts:50]: create classes for these components.

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx (hunk 1 above)
Comment (AtishayMsft) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:14]: processed style set is not needed. palette is v8. replace it with v9 tokens

Comment (AtishayMsft) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:14]: only line 14 needs to be removed. I have highlighted it in the comment.

Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:14]: Okay sure

Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:14]: > only line 14 needs to be removed. I have highlighted it in the comment.

Okay, I thought it was for all the imports

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.types.ts (hunk 2 above)
Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.types.ts:2]: For this, basically all imports from react/lib/utilities need to be removed, right?

Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.types.ts:2]: In this, IRenderFunction is being used to render the chart

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**